index.bs

```diff
@@ -2857,16 +2881,31 @@ To <dfn noexport>get an OS-registration URL from a header list</dfn> given a
 1. If |value| is not a [=string=], return null.
 1. Return the result of running the [=URL parser=] on |value|.
 
+"<code><dfn>Attribution-Reporting-Support</dfn></code>" is a Dictionary
+Structured Header set on a [=request=] that indicates which registrars, if
+any, the corresponding [=response=] can use. Its values are not specified and
```

Comment:
They are booleans right now (always `true`), both in the spec and implementation, but we don't necessarily want reporting origins to interpret the values for maximum compatibility with a hypothetical change in which the values do have meaning. If you think that's overkill, then I can specify them, though we may want to consider just using a List Structured Header instead of a Dictionary one whose keys are permanently `true` (and where `false` would have to be given a meaning).

Reply:
OK that's fine.